### DISCUSSION TITLE
Welcome to dswj Discussions!

### DISCUSSION THREAD

**Discussion by paTait** [paTait/dswj#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paTait** [comment]
Hi Everyone,
I am planning on updating the dswj repository to reflect the current state of the Julia (v1.5.x) and its package ecosystem over the first quarter of 2021. Please feel free to make pull requests and use this discussion forum to make comments or ask questions.
Peter

**Reply by Dainiu-Zhang** [reply]
Could you please provide the chp7_ranger.R code in chp7, I can’t find it😭

**Comment by paTait** [comment]
@TY-Cheng - thank you for the pull requests. I will review them in the coming days.
Peter
